ios/iden3CoreSDK/Iden3Identity.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  Iden3Identity.swift
+//  iden3CoreSDK
+//
+//  Created by Iden3 on 12/06/2020.
+//  Copyright © 2020 Iden3. All rights reserved.
+//
+
+import Foundation
+import Iden3mobile
+
+public final class Iden3Identity {
+    
+    // MARK: - Properties
+    
+    private var identity: Iden3mobileIdentity
+    
+    // MARK: - Initializers
+       
+    init(identity: Iden3mobileIdentity) {
+        self.identity = identity
+    }
+    
+    /// Stops the Iden3 identity.
+    ///
+    public func stopIdentity() {
+        identity.stop()
+    }
+    
+    /// Request a claim from an issuer.
+    ///
+    /// - parameters:
+    ///   - issuerUrl: Url of the issuer
+    ///   - data: data needed by the issuer for requesting a claim.
+    ///   - ticketDelegate: Delegate of tickets received after requesting the claim.
+    ///
+    ///   return Iden3Ticket.
+    ///
+    public func requestClaim(issuerUrl: String, data: String, ticketDelegate: Iden3TicketDelegate?) throws -> Iden3Ticket? {
+        if (ticketDelegate == nil) {
+            do {
+                return Iden3Ticket(try identity.requestClaim(issuerUrl, data: data))
+            } catch {
+                return nil
+            }
+        } else {
+            identity.requestClaim(withCb: issuerUrl, data: data, c: nil)
+        }
+        return nil
+    }
+    
+    /// Sends a credentialValidity build from the given credentialExistance to a verifier.
+    ///
+    /// - parameters:
+    ///   - verifierUrl: url of the verifier
+    ///   - credentialId: credential id needed by the verifier for proving a claim.
+    ///   - withZKProof: Boolean that indicates if zero-knowledge proof shoud be used or not.
+    ///   - proveClaimDelegate: Delegate of proofs received after proving a claim.
+    ///
+    ///   return Bool should be true if the verifier accepted the prove as valid
+    ///
+    public func proveClaim(verifierUrl: String, credentialId: String, withZKProof: Bool, proveClaimDelegate: Iden3ProveClaimDelegate?) throws -> Bool {
```

Comment:
Same comment I did with the Android equivalent method: I will consider only using the "withCb" variant.

Reply:
done